app/uk/gov/hmrc/bindingtariffclassification/repository/SearchMapper.scala

```diff
@@ -69,13 +72,30 @@ class SearchMapper {
     }
   }
 
-  private def notEqualFilter(field: String): JsObject = {
-    Json.obj("$ne" -> field)
+  private def numberStartingWith(value: String): JsObject = {
+    Json.obj( regexFilter(s"^$value\\d*") )
```

Comment:
I assume we store only digits in mongo

Reply:
I also assume the front-end does the validation on the commodity code (it should be a non-empty string with digits only, with a length between 6 and 22 characters)

Reply:
It could be an empty string if it’s in progress but that should be filtered out so should be ok.
UI only allows numbers so should be ok